language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb

```diff
@@ -50,12 +50,12 @@
   },
   {
    "cell_type": "markdown",
-   "id": "2fbcb31c",
+   "id": "45633f2d-d69e-47a3-9ddc-c93fa91caba7",
    "metadata": {},
    "source": [
     "| | |\n",
     "|-|-|\n",
-    "|Author(s) | [Sanchit Latawa](https://github.com/slatawa) |"
+    "|Author(s) | [Sanchit Latawa](https://github.com/slatawa) , Lalit Kumar Digala"
```

Comment:
Can you add the other author's github page?

Reply:
Added